test/stream.js

```diff
@@ -20,12 +18,18 @@ const file = new Vinyl({
   contents: fs.createReadStream(filePath),
 });
 
-test('should error if contents is a stream', function(t) {
-  const stream = task();
-  stream.on('error', function(err) {
-    t.ok(err instanceof PluginError, 'not an instance of PluginError');
-    t.end();
+describe('stream', function() {
+  it('should error if contents is a stream', function(done) {
+    pipe(
+      [from([file]), task()],
+      (err) => {
+        try {
+          expect(err).toBeInstanceOf(TypeError);
```

Comment:
This is the main difference compared to the old tests: I got a `TypeError` instead of a `PluginError`. I'll check why.

Reply:
It's actually because you used `from` instead of `from.obj` so the stream thought it was operating in buffer mode instead of object mode.